### ISSUE TITLE
# TLDR

### ISSUE THREAD

**Issue by lyjmry** [filecoin-project/FIPs#1042]
# TLDR

- Filecoin is a thriving ecosystem with an exciting growth trajectory ahead. However, **FIL’s current tokenomics, in particular its high FDV relative to MC, are perceived to be challenging for new investors** to enter into the ecosystem. 

- With the aim of making Filecoin more aligned with how the market assesses the attractiveness of crypto networks, **we propose that the 300M of FIL mining reserve be burned (15% of the 2B maximum supply)**. From the token holders’ perspective, the burn would optically reduce the dilution overhang, enhancing the perceived stability of Filecoin’s token economy. 

# Motivation

Fully Diluted Value (FDV) is a common metric the market looks at to evaluate networks and see how much upcoming dilution token holders might incur - FDV is measured by extrapolating the market cap of the network if the full theoretical supply of tokens were to be released.

Market Cap (MC) is a metric that measures the total value of a crypto asset based on its current price and current circulating supply (rather than its maximum, future total supply). Large gaps between FDV and MC can be a red flag because it signals to investors that their token holdings will be diluted with inflation as the current circulating supply expands into its maximum, future total supply. As a result, a low MC to FDV ratio - while it does not give any information about the rate of future token inflation - likely dissuades investors from considering an investment in the underlying crypto asset in the first place. We note that there has been a positive correlation between the MC/FDV ratio and token price returns over different time periods - which potentially provides empirical backing to our hypothesis of investor bias against low MC/FDV tokens. 

<img width="266" alt="image" src="https://github.com/filecoin-project/community/assets/21000523/0e5ea108-4466-4760-bc15-05e6c121ba7a">

As it currently stands, most people use Filecoin’s 2 billion total supply number to calculate Filecoin's FDV. This number includes tokens that are not guaranteed to enter the total supply - including the mining reserve and the full 770m baseline minting rewards (which would require reaching 1YiB of RBP to release). 

<img width="507" alt="image" src="https://github.com/filecoin-project/community/assets/21000523/d75d91d1-7126-4a6d-8556-6d75f1abfa13">

Source: [Token Unlocks](https://token.unlocks.app/filecoin)

The result is that Filecoin has the 10th lowest MC/FDV ratio in the top 100 tokens by market cap, with a large perceived overhang of future dilution. The average and median MC/FDV for the top 100 cohort are much higher at 76% and 89%, respectively. Although MC/FDV is a time-agnostic metric that does not account for the rate of token inflation, feedback from the investment community indicates that a high FDV relative to the MC is indeed offputting.

<img width="437" alt="image" src="https://github.com/filecoin-project/community/assets/21000523/65b70ad2-36e0-42db-8519-0c734d5a3cce">

Source: [Artemis.xyz](https://artemis.xyz)

# Proposal: Burn the Mining Reserve (300M/2B FIL tokens) 

The [goal](https://spec.filecoin.io/systems/filecoin_token/token_allocation/) of the Mining Reserve was to hold back additional tokens to be used to incentivize future types of mining. Now that we’re several years into the network (and crypto as an industry has evolved), it seems clear that there are new mechanics/solutions we can lean into to incentivize new types of actions, rather than relying on creating new sources of dilution.

3 main observations:

(a) **Evolving network structure**: Initially, the token design assumed there would be newer forms of mining that need to use FIL as part of the incentive mechanics. Today, it seems the direction the ecosystem is going towards is with IPC and subnets vs having an enshrined set of new mining.

(b) **Evolving network alignment strategy**: The original token design determined that new forms of mining should have a separate token pool and inflation schedule from the “core” inflation schedule from baseline+simple minting. Today, it seems like the models we should pursue might look closer to Eigenlayer/staking FIL for securing IPC subnets (and even new tokens here) vs creating new sources of FIL emissions. 

(c) **Addressing the high FDV overhang**: The token design assumed the community would be willing to accept 300 million FIL in additional inflation (as would be required to pass a FIP) and that the meme of 2 billion FIL would be stronger than the meme of 1.7 billion FIL. However, given it takes a FIP to introduce the mining reserve into circulation (which would be the exact same process as creating a FIP to increase the supply to over 2 billion), it seems like we’re unnecessarily hampering the ecosystem by allowing these tokens to be counted in the FDV. If the community wants to FIP in more inflation in the future, they are welcome to propose such a change - but it’s unclear why we should allow the overhang of potential inflation to create a weight on the ecosystem today.

_Note: Various teams are putting a lot of work into driving the adoption of paid deals and reducing friction for FEVM builders - and this is not a substitute for that. However, if we don't address the FDV issue - the hurdle for Filecoin to get credit for any progress on the fundamentals will be substantially higher._

_Originally posted by @jnthnvctr in https://github.com/filecoin-project/FIPs/discussions/1030_

**Comment by lyjmry** [comment]
I oppose this proposal because the shares of investment institutions, teams, and foundations have not been reduced. If it is unfair to destroy the mining reserve, I hope that 300 million FIL will be added to the existing mining benchmark, because mining returns are now extremely declining, hindering the growth of the network. Network growth is beneficial to the long-term development of the network.

**Comment by luckyparadise** [comment]
Hi @lyjmry thanks for sharing your thoughts on this topic. I can see that you also responded to the discussion thread directly. Is there any reason you have opened your response here too as an issue? 

You are more than welcome to start a separate discussion on this topic in the discussions tab and seek additional input from community members. 

If there is no other reason for opening this as an issue, I recommend closing it.
